packages/block-editor/src/components/copy-handler/index.js

```diff
@@ -198,6 +198,7 @@ export function useClipboardHandler() {
 				if ( shouldHandleWholeBlocks && ! expandSelectionIsNeeded ) {
 					removeBlocks( selectedBlockClientIds );
 				} else {
+					event.target.ownerDocument.activeElement.contentEditable = false;
```

Comment:
This is actually a bug introduced in #40098. When using `__unstableDeleteSelection`, contenteditable needs to be turned off first (see other instances). Ideally we should move this logic to writing flow and build in clipboard handling there.